kubernetes_deploy/staging/deployment.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-fala
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: laa-fala-app
+  template:
+    metadata:
+      labels:
+        app: laa-fala-app
+    spec:
+      containers:
+      - image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/laa-get-access/laa-fala:ecr-docker-registry
+        name: app
+        env:
+        - name: DEBUG
+          value: "True"
+      - name: nginx
+        image: nginx:1.14
+        volumeMounts:
+        - name: nginx-conf
+          mountPath: /etc/nginx/conf.d
```

Comment:
It would probably be easier to use nginx if we had a `Dockerfile.nginx` file in the repository and a `docker-compose.yml` file to build/use them together. What do you think?

Reply:
Yeah, whilst we're in the development stage, definitely make sense for faster feedback.

I'll pull the nginx related code bits out of this pull requests's branch.